signac_dashboard/views.py

```diff
@@ -53,6 +51,15 @@ def jobs_list(dashboard):
     return dashboard._render_job_view(default_view="list")
 
 
+def project_info(dashboard):
+    g.project = dashboard.project
+    project_title = dashboard.project.config["project"]
```

Comment:
The `next` branch of signac fails here, because we no longer have project titles. However, there are a lot of things we'll need to fix in the future so don't worry about compatibility with signac `next` yet.

Reply:
Yeah I'll make a new issue to track these but will resolve these conversations for now